### ISSUE TITLE
Head hitbox becomes almost unhittable when looking down

### ISSUE THREAD

**Issue by Safewood** [ValveSoftware/halflife#329]
When you're shooting at a player who looks downwards the bullets most of the time just goes right through.

Example: http://www.youtube.com/watch?feature=player_detailpage&v=Jskxvlf5SlU#t=134s

**Comment by Maxi605** [comment]
@alfred-valve can you close this? it's the same issue that has been talked here https://github.com/ValveSoftware/halflife/issues/2154

**Comment by RauliTop** [comment]
> @alfred-valve can you close this? it's the same issue that has been talked here #2154

alfred?? Probably @kisak-valve

**Comment by Maxi605** [comment]
> > @alfred-valve can you close this? it's the same issue that has been talked here #2154
> 
> alfred?? Probably @kisak-valve

Yeah, my bad, i got confused for a sec and didn't remember his name haha

**Comment by agrastiOs** [comment]
@kisak-valve should be closed, duplicate of the better known #2154.
